nautobot/extras/datasources/git.py

```diff
@@ -742,14 +742,15 @@ def refresh_code_from_repository(repository_slug, consumer=None, skip_reimport=F
             logger.debug("Unloading module %s", module_name)
             if module_name in app.loader.task_modules:
                 app.loader.task_modules.remove(module_name)
-            del sys.modules[module_name]
+            if module_name in sys.modules:
```

Comment:
On inspection, this check actually is redundant since we're explicitly iterating over `sys.modules` but it's not harmful either.